### ISSUE TITLE
Warrior Priest Purify With Cleansing Power Tactic

### ISSUE THREAD

**Issue by sybreal** [WarEmu/WarBugs#10373]
My Purify ability works fine, and as intended without the tactic. However, with the tactic it starts to mess up. 

When I cleanse myself, only I get cleansed.
When I cleanse someone in my party, everyone in my party except for me gets cleansed.
I cannot cleanse anyone outside of my party, either in my warband or out of it.

I have tested this multiple times with random enemy ailments and such and more accurately with the Keep Lord debuff.

**Comment by wargrimnir** [comment]
Intentional change from live. 
https://www.returnofreckoning.com/forum/viewtopic.php?f=52&t=8768#p92913

**Comment by Sinrman** [comment]
While this was closed by Wargrimnir I ask that is be reopened on the 3rd point from the original poster.  With the cleansing power tactic slotted you cannot cleanse anyone not in your party.  I tried running solo with this tactic and you are unable to cleanse any hex or curse on someone other than you.  In a warband you cannot cleanse anyone outside of your group even when targeting them specifically.
